taxonomies/food/categories.txt

```diff
@@ -94787,6 +94774,7 @@ intake24_category_code:en: PTNS
 pnns_group_2:en: Cereals
 wikidata:en: Q178
 wikipedia:en: https://en.wikipedia.org/wiki/Pasta
+opposite:en: en:pasta-dishes
```

Comment:
How is this the _opposite_ of pasta dishes? Did you mean to use `incompatible_with`?
```suggestion
incompatible_with:en: en:pasta-dishes
```